dependencies.yaml

```diff
@@ -140,34 +141,41 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - pre-commit
-  cudatoolkit:
+  cuda_version:
     specific:
       - output_types: conda
         matrices:
           - matrix:
               cuda: "11.2"
             packages:
               - cuda-version=11.2
-              - cudatoolkit
           - matrix:
               cuda: "11.4"
             packages:
               - cuda-version=11.4
-              - cudatoolkit
           - matrix:
-              cuda: "11.5"
+              cuda: "11.6"
             packages:
-              - cuda-version=11.5
-              - cudatoolkit
+              - cuda-version=11.6
```

Comment:
Just checking did want to change this from 11.5 to 11.6?

Reply:
no sorry, that was an oversight. Reverted in https://github.com/rapidsai/cucim/pull/671/commits/1502e2039a64e993b27021d8e62f9ba88a32ffea.